src/App.js

```diff
@@ -4,35 +4,21 @@ import PropTypes from 'prop-types';
 import Banner from './features/Banner';
 import Movies from './features/Movies';
 import Footer from './shared/Footer';
+// import our context
+import { ProjectProvider } from './store/ProjectContext';
 import './App.scss';
 
 
 export default class App extends Component {
-	constructor(props){
-		super(props);
-		this.state = {
-			isHidden : true
-		}
-		this.handleToggleIsHidden = this.handleToggleIsHidden.bind(this);
-	}
-
-		handleToggleIsHidden() {
-			this.setState({ isHidden: !this.state.isHidden });
-		}
 	render() {
-		const {isHidden} = this.state;
 		return (
-			<div>
-				<Banner 
-					isHidden={isHidden} 
-					handleToggleIsHidden={this.handleToggleIsHidden}
-				/>
-				<Movies 
-					isHidden={isHidden} 
-					handleToggleIsHidden={this.handleToggleIsHidden}
-				/>
-				<Footer />
-			</div>
+			<ProjectProvider>
+				<div>
```

Comment:
what do you need empty div for?
if it doesn't do anything, consider using Fragment instead

Reply:
Switched over to <></> instead. It was there in case I needed to apply some generic styles to App.js.